test/refactor_nrepl/integration_tests.clj

```diff
@@ -206,6 +207,14 @@
           result (remove keyword? response)]
       (is (= 3 (count result)) (format "expected 3 results but got %d" (count result))))))
 
+(deftest find-local-in-namespaced-destructuring
```

Comment:
We could do quite simply:

```clj
(when (modern? *clojure-version*)
  (deftest ...)
```

It seems worth it since other than this deftest there seemingly isn't another strong reason to drop 1.8 support.

I think keeping 1.8 support is a good idea. Many modern projects still use 1.8 or such by default, and 1.10 will be defined in a separate profile. For those cases, refactor-nrepl will run in an 1.8 environment by default.

Reply:
Might as well drop it now that we are bumping our major version.  

As I mentioned in the commit message people can still run 1.8 if they want to, as the clojure version is managed by the user.

Reply:
There's the nuance that dropping 1.8 from CI would mean we would no longer detect incompatibilities in advance.